tests/tests.ts

```diff
@@ -1,102 +1,110 @@
-        assert(testUtils.artifactoryUrl, 'Tests are missing environment variable: ADO_ARTIFACTORY_URL');
-        assert(testUtils.artifactoryUsername, 'Tests are missing environment variable: ADO_ARTIFACTORY_USERNAME');
-        assert(testUtils.artifactoryPassword, 'Tests are missing environment variable: ADO_ARTIFACTORY_PASSWORD');
+        assert.ok(TestUtils.artifactoryUrl, 'Tests are missing environment variable: ADO_ARTIFACTORY_URL');
+        assert.ok(TestUtils.artifactoryUsername, 'Tests are missing environment variable: ADO_ARTIFACTORY_USERNAME');
+        assert.ok(TestUtils.artifactoryPassword, 'Tests are missing environment variable: ADO_ARTIFACTORY_PASSWORD');
 
-        testUtils.initTests();
-        jfrogUtils = require('artifactory-tasks-utils');
-        repoKeys = testUtils.getRepoKeys();
+        TestUtils.initTests();
+        repoKeys = TestUtils.getRepoKeys();
     });
 
-    beforeEach(() => {
+    beforeEach((): void => {
         tasksOutput = '';
     });
 
-    after(() => {
-        testUtils.cleanUpAllTests();
+    after(function(): void {
+        this.timeout(120000); // 2 minute timer for the after hook only.
+        TestUtils.cleanUpAllTests();
     });
 
-    describe('Unit Tests', () => {
+    describe('Unit Tests', (): void => {
         console.log('OS:', os.type());
-        runTest('Mask password', () => {
-            let oldPassword = process.env.ADO_ARTIFACTORY_PASSWORD;
+        runSyncTest('Mask password', (): void => {
+            const oldPassword: string = process.env.ADO_ARTIFACTORY_PASSWORD || '';
             process.env.ADO_ARTIFACTORY_PASSWORD = 'SUPER_SECRET';
-            let retVal;
+            let retVal: string = '';
             try {
                 jfrogUtils.executeCliCommand(
                     'jfrog rt del ' +
                         repoKeys.repo1 +
                         '/' +
                         ' --url=' +
-                        jfrogUtils.quote(process.env.ADO_ARTIFACTORY_URL) +
+                        jfrogUtils.quote(process.env.ADO_ARTIFACTORY_URL || '') +
                         ' --user=' +
-                        jfrogUtils.quote(process.env.ADO_ARTIFACTORY_USERNAME) +
+                        jfrogUtils.quote(process.env.ADO_ARTIFACTORY_USERNAME || '') +
                         ' --password=' +
                         jfrogUtils.quote('SUPER_SECRET'),
-                    testUtils.testDataDir,
+                    TestUtils.testDataDir,
                     ['']
                 );
             } catch (ex) {
                 retVal = ex.toString();
             }
+            assert.ok(retVal !== '', 'An exception should have been caught');
             process.env.ADO_ARTIFACTORY_PASSWORD = oldPassword;
-            assert(!retVal.toString().includes('SUPER_SECRET'), 'Output contains password');
+            assert.ok(!retVal.includes('SUPER_SECRET'), 'Output contains password');
         });
 
-        runTest(
+        runAsyncTest(
```

Comment:
```suggestion
        runSyncTest(
```
This function can be sync. See comment in line 82.
If you agree with this change, you can delete runAsyncTest and rename runSyncTest => runTest